app/controllers/movies_controller.rb

```diff
@@ -63,7 +63,9 @@ def dislike
   # GET /movies/1.json
   def show
     # @review = @user.reviews.where(movie_id: @movie.id).order("created_at DESC")
-    @review = Review.where(user_id: @user, movie_id: @movie.id).order(created_at: :desc).paginate(per_page: 15, page: params[:page])
```

Comment:
@sdsunjay if you're not using `@reviews` you could do `@user.reviews.find_by(movie_id: @movie.id)`

Reply:
Why would I do this?